code/src/GridAPI/Filter.ts

```diff
@@ -17,40 +17,59 @@ namespace GridAPI.Filter {
     }
 
     /**
-     *
+     * Funtion that perform a Search at a given GridID by a given value
      *
      * @export
      * @param {string} gridID
      * @param {string} searchedValue
-     * @returns {*}  {void}
+     * @returns {*}  {string} Return Message Succes or message of error info if it's the case.
```

Comment:
@joselrio Typo in Success. Should be something like "Return the success message or error message if any occurs"

Reply:
Done!